### PR TITLE
Fix set murdered

### PR DESCRIPTION
**Pull request by ZiadMontaser**
### Description
Fix set murdered
<!--

### CODE REVIEW

**Comment by js6pak** [comment]
Wouldn't this break game state, anticheat would think that player is still alive? And this really should be done other way around + with dead/impostor checks

**Comment by ZiadMontaser** [comment]
are we good now

File: src/Impostor.Api/Net/Inner/Objects/IInnerPlayerControl.cs [hunk 1]
```diff
@@ -111,5 +111,13 @@ public interface IInnerPlayerControl : IInnerNetObject
         /// </summary>
         /// <returns>Task that must be awaited.</returns>
         ValueTask SetMurderedAsync();
```
Comment (AlexejheroYTB) [inline src/Impostor.Api/Net/Inner/Objects/IInnerPlayerControl.cs:113]: Remove `SetMurderedAsync`

File: src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.Api.cs [hunk 2]
```diff
@@ -103,5 +106,19 @@ public async ValueTask SetMurderedAsync()
             writer.Write((byte)NetId);
             await _game.FinishRpcAsync(writer);
         }
+
+        public async ValueTask SetMurderedByAsync(IClientPlayer impostor)
+        {
+            var impostorNetId = impostor.Character.NetId;
```
Comment (AlexejheroYTB) [inline src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.Api.cs:112]: Check if the `impostor` is actually an impostor, check if they're not dead, and also check if the target player is not dead.

**Review by AlexejheroYTB — CHANGES_REQUESTED** [review]

**Comment by AlexejheroYTB** [comment]
Edit the description of the PR to include `closes #183`

**Pull request merged**